CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Versions
 
-## x.x.x
+## 1.1.2
```

Comment:
This should be at least a minor version bump, b/c we are adding new APIs. 1.2.0?